polyfill/test/yearmonth.mjs

```diff
@@ -349,7 +349,7 @@ describe('YearMonth', () => {
   describe('YearMonth.with()', () => {
     it('throws on bad disambiguation', () => {
       ['', 'CONSTRAIN', 'balance', 3, null].forEach((disambiguation) =>
-        throws(() => YearMonth.from(2019, 1).with({ month: 2 }, { disambiguation }), RangeError)
+        throws(() => YearMonth.from({ year: 2019, month: 1 }).with({ month: 2 }, { disambiguation }), RangeError)
```

Comment:
Thank you TypeScript!

Reply:
TS would have caught this, but I haven't yet tried checking `YearMonth` tests with TS. Instead, it was adding the `GetObject` assertion exception that uncovered it.